### PR TITLE
feat: add `filterItemClass` prop to `HierarchicalFilter`

### PR DESCRIPTION
**Pull request by diegopf**
EMP-120

Adds a new dynamic prop to be able to style the filter element itself.

### CODE REVIEW

File: packages/x-components/src/x-modules/facets/components/filters/hierarchical-filter.vue [hunk 1]
```diff
@@ -18,7 +18,7 @@
         <button
           @click="clickFilter"
           :aria-checked="filter.selected.toString()"
-          :class="cssClasses"
+          :class="[cssClasses, filterItemClass]"
```
Comment (tiborux) [inline packages/x-components/src/x-modules/facets/components/filters/hierarchical-filter.vue:21]: should we expose this class in the slot?

**Review by tiborux — APPROVED** [review]

**Comment by tiborux** [comment]
gj

File: packages/x-components/src/x-modules/facets/components/filters/hierarchical-filter.vue [hunk 2]
```diff
@@ -82,7 +83,10 @@
   @Component({
     name: 'HierarchicalFilter',
     components: { FiltersList, RenderlessFilter },
-    mixins: [xComponentMixin(facetsXModule), dynamicPropsMixin(['childrenFiltersClass'])]
+    mixins: [
+      xComponentMixin(facetsXModule),
+      dynamicPropsMixin(['childrenFiltersClass', 'filterItemClass'])
```
Comment (mnavarroespinosa) [inline packages/x-components/src/x-modules/facets/components/filters/hierarchical-filter.vue:88]: Could you add the new property `filterItemClass` to the docs below?

**Review by mnavarroespinosa — CHANGES_REQUESTED** [review]

**Review by mnavarroespinosa — APPROVED** [review]

**Review by tiborux — APPROVED** [review]

**Pull request merged**